website/pages/en/network/curating.mdx

```diff
@@ -81,17 +87,20 @@ Finding high-quality subgraphs is a complex task, but it can be approached in ma
 - Curators can use their understanding of a network to try and predict how an individual subgraph may generate a higher or lower query volume in the future
 - Curators should also understand the metrics that are available through The Graph Explorer. Metrics like past query volume and who the subgraph developer is can help determine whether or not a subgraph is worth signalling on.
 
-### 3. What’s the cost of updating a subgraph?
+### 3. What’s the cost of upgrading a subgraph?
 
-Migrating your curation shares to a new subgraph version incurs a curation tax of 1%. Curators can choose to subscribe to the newest version of a subgraph. When curation shares get auto-migrated to a new version, Curators will also pay half curation tax, ie. 0.5%, because updating subgraphs is an on-chain action that costs gas.
+Migrating your curation shares to a new subgraph version incurs a curation tax of 1%. Curators can choose to subscribe to the newest version of a subgraph. When curator shares get auto-migrated to a new version, Curators will also pay half curation tax, ie. 0.5%, because upgrading subgraphs is an on-chain action that costs gas.
 
-### 4. How often can I update my subgraph?
+### 4. How often can I upgrade my subgraph?
```

Comment:
I think we use `update` for new versions ("upgrade" is used to reference moving to the network)

Reply:
good idea to change that